CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 ## Unreleased
 - Replace `Symplify\CodingStandard\Sniffs\Naming\[AbstractClassNameSniff, ClassNameSuffixByParentSniff, InterfaceNameSniff and TraitNameSniff]` with equivalent versions backported to this repository.
+- Drop PHP 7.2 support.
```

Comment:
pls label as BC

Reply:
Dependency change is not BC break (library public API was not changed). See https://sebastiandedeyne.com/composer-semver-and-underlying-dependency-changes/

However, this whole will be part of php-coding-standard 3.0, so it will effectively be in next major release.